scripts/topology/test_topology.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import numpy as np
+
+import fourier_to_ryckaert_bellemans as ftrb
+import ryckaert_bellemans_to_fourier as rbtf
```

Comment:
Not sure if this is the best way to test the scripts. This is just a proof of concept to have some tests to run. Ideally the tests are located in their own 'tests' directory in the root, but this makes testing the scripts harder.
Fell free to replace by better suited method in the future.